### PR TITLE
[Add] Guide to Python wheels

### PR DESCRIPTION
**Pull request by tonguegrease**
Fixing https://github.com/openvinotoolkit/openvino/issues/22017 by adding a guide to Python wheels. Hopefully I did everything correctly. This is one of my first contributions so I apologize in advance for any mistakes I may have made. Thank you for allowing me the opportunity to work on this!

Details:
Made a guide to Python wheels by adding:
An introduction
Elaborating the description
Building section
Locating section
Setup section
Workflow section

Tickets:
ticket-id

### CODE REVIEW

File: src/bindings/python/wheel/guide-to-wheels.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+To build Python wheels during the project build process, make sure to include the following options when running the cmake command:
+  1. bash
+  2. Copy code
+  3. cmake -DENABLE_PYTHON=ON -DENABLE_WHEEL=ON ..
+
+This ensures that the build process includes the Python components and generates the necessary wheel files.
+
+Locating Python Wheels
+#######################
+After building the project, you can find the generated Python wheels under the **openvino_install_dir/tools** directory.
```
Comment (jiwaszki) [inline src/bindings/python/wheel/guide-to-wheels.rst:34]:
```suggestion
After building the project, you can find the generated Python wheels under the **openvino/build/wheels/** directory.
```
I think this is the easiest way to locate them after building the project and this way should be advised.

File: src/bindings/python/wheel/guide-to-wheels.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+`openvino-dev <https://github.com/openvinotoolkit/openvino/blob/master/tools/openvino_dev/setup.py>`__: Additional development tools and utilities.
+
+Building Python Wheels
+#######################
+To build Python wheels during the project build process, make sure to include the following options when running the cmake command:
+  1. bash
+  2. Copy code
+  3. cmake -DENABLE_PYTHON=ON -DENABLE_WHEEL=ON ..
+
+This ensures that the build process includes the Python components and generates the necessary wheel files.
```
Comment (jiwaszki) [inline src/bindings/python/wheel/guide-to-wheels.rst:30]: I would also add info about additional effects of this building pratice:
* That wheels regeneration might need require to delete `wheels` folder before running `make` command again.
* After getting new wheels it is required to reinstall them, i.e. running `pip uninstall openvino` and `pip install wheels/openvino[...]` as effects won't be visible instantly.

File: src/bindings/python/wheel/guide-to-wheels.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+Python Environment Setup
++++++++++++++++++++++++++
+Ensure that you have the necessary Python wheels installed in your environment:
+  1. bash
+  2. Copy code
+  3. pip install openvino
+  4. pip install openvino-dev
+
+Workflow Differences
```
Comment (jiwaszki) [inline src/bindings/python/wheel/guide-to-wheels.rst:49]: I would also include the case when both pip and PYTHONPATH is set together. This results in conflicts which can be describe in this guide.

**Review by jiwaszki — CHANGES_REQUESTED** [review]
@tonguegrease thanks for this PR! Here is my feedback on it.

CC @kblaszczak-intel please take a look here from documentation point of view, thanks!

**Comment by mlukasze** [comment]
hey @tonguegrease :)
Will you have a time to finish a PR?

**Comment by tonguegrease** [comment]
Hey there! Apologies for the late reply. Unfortunately, I have too many things going on in my life at the moment and won't have time to finish this.On Mar 26, 2024, at 2:08 AM, Michal Lukaszewski ***@***.***> wrote:﻿
hey @tonguegrease :)
Will you have a time to finish a PR?

**Comment by mlukasze** [comment]
thank you for your time any way!

**Pull request closed without merging**